po/pl.po

```diff
@@ -1535,25 +1561,24 @@ msgstr "Srebrny, plastyczny metal"
 
 #: ../libelemental/data.cc:83
 msgid "Named after Strontian, Scotland."
-msgstr ""
+msgstr "Nazwany od miejscowości Stronthian w Szkocji"
```

Comment:
"Stronthian" should be "Strontian". This was an error in the Polish Wikipedia entry for Strontium (which I corrected).